cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -269,6 +269,15 @@ public void scanApplications(Cursor cursor,
         StoreDefinition.AppMetadataStore.APPLICATION_DATA_FIELD);
   }
 
+  public List<ApplicationMeta> getAllApplications() throws IOException {
```

Comment:
where is this function called?

Reply:
Forgot to remove this from previous approach. I've renamed my new function to use this name since its more descriptive